homework/SeleniumAdvanced/EpamSiteNavigateTests.cs

```diff
@@ -0,0 +1,79 @@
+﻿using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium.Interactions;
+using OpenQA.Selenium.Support.UI;
+using System.Collections.Generic;
+using System.Runtime.Intrinsics.X86;
+
+namespace SeleniumAdvanced
+{
+    [TestFixture]
+    public class EpamSiteNavigateTests
+    {
+        private IWebDriver _driver;
+        private WebDriverWait _waiter;
+        private Actions _action;
+        private IJavaScriptExecutor? _javaScriptExecutor;
+        private const string link = "https://www.epam.com";
+        
+
+        [SetUp]
+        public void InicializeBrowserAndWaits()
+        {
```

Comment:
suggestion: it is better to split setups :)
Single Responsibility principle in SOLID :)